app/controllers/finders_controller.rb

```diff
@@ -30,11 +30,15 @@ def show
       format.atom do
         if content_item.is_redirect?
           redirect_to_destination
-        elsif finder.atom_feed_enabled?
-          expires_in(ATOM_FEED_MAX_AGE, public: true)
-          @feed = AtomPresenter.new(finder, results, facet_tags)
         else
-          render plain: 'Not found', status: :not_found
+          expires_in(ATOM_FEED_MAX_AGE, public: true)
+          @feed = AtomPresenter.new(finder_for_feed, results_for_feed, facet_tags_for_feed)
+
+          # clobber @finder so that the FinderAnswerABTest doesn't
+          # trigger another call to search-api when it accesses this
+          # variable - all it wants to know is if this is the eu exit
+          # finder.
+          @finder = @finder_for_feed
```

Comment:
As `finder_for_feed` is memoized, could you pass the method rather than the variable here?